policies/continuous_feedback.md

```diff
@@ -1,108 +1,126 @@
-#Continuous Feedback
+# Continuous Feedback
 
 The techniques we use to deliver software tell us to value things like regular, open communication and fast feedback loops. We’re aiming to cultivate a culture where individual performance can be evaluated and coached equally regularly.
 
 There is a growing trend for organisations to throw away the annual / bi-annual review, and move to more informal and more frequent check-ins. Adobe is one such business, and they’ve published much of their collateral online, on which this structure is based.
 
 In contrast to how feedback is delivered at present, which is a largely ‘top-down’ approach, we would like to shift to a model where each team member is ultimately responsible for pro-actively soliciting feedback on their own performance.
 
-## How it works
+ - [How does it work?](#how-does-it-work)
+ - [How often should I have a session?](#how-often-should-i-have-a-session)
+ - [What kind of feedback should I give?](#what-kind-of-feedback-should-i-give)
+ - [How should I receive feedback?](#how-should-i-receive-feedback)
+ - [What should I consider during my CF session?](#what-should-i-consider-during-my-cf-session)
 
-Our continuous feedback sessions involve 2 people, yourself and a facilitator. The facilitator will either be a colleague or a director. Their role is to help you make sense of your feedback and to help you track your progress against goals you've set yourself.
+## How does it work?
+
+Our continuous feedback sessions involve 2 people, yourself and a facilitator. The aim of a continuous feedback session is to:
+
+ - Receive and acknowledge feedback
+ - Setting and reviewing longer term goals
+ - Setting and reviewing short term goals
+
+The facilitator will either be a colleague or a director. Their role is to help you make sense of your feedback and to help you track your progress against goals you've set yourself.
```

Comment:
Don't like the fact that the wording "colleague or a director" implies at first glance that directors are not colleagues